frontend/src/components/DatePicker/components/DatePickerWrapper.tsx

```diff
@@ -4,40 +4,37 @@ import { useDatePicker } from '../DatePickerContext'
 
 import { DatePickerInput } from './DatePickerInput'
 
-export const DatePickerWrapper = forwardRef<object, 'input'>(
-  ({ children }, ref) => {
-    const {
-      disclosureProps,
-      initialFocusRef,
-      closeCalendarOnChange,
-      isMobile,
-    } = useDatePicker()
-
-    if (isMobile) {
-      return (
-        <Flex>
-          <DatePickerInput ref={ref} />
-          {children}
-        </Flex>
-      )
-    }
+export const DatePickerWrapper = forwardRef<
+  { isHighContrast?: boolean },
+  'input'
+>(({ isHighContrast, children }, ref) => {
+  const { disclosureProps, initialFocusRef, closeCalendarOnChange, isMobile } =
+    useDatePicker()
 
+  if (isMobile) {
     return (
       <Flex>
-        <Popover
-          placement="bottom-start"
-          isLazy
-          initialFocusRef={initialFocusRef}
-          closeOnBlur={closeCalendarOnChange}
-          returnFocusOnClose={false}
-          {...disclosureProps}
-        >
-          <PopoverAnchor>
-            <DatePickerInput ref={ref} />
-          </PopoverAnchor>
-          {children}
-        </Popover>
+        <DatePickerInput ref={ref} isHighContrast={isHighContrast} />
```

Comment:
This is not required since the provider is the one that passes the `isHighContrast` prop and this is never used.

Reply:
fixed in https://github.com/opengovsg/FormSG/pull/8251/commits/fef5d89dbce7f2cfc768db8517a5ca4bde276a93